common/errors/exit_codes.go

```diff
@@ -0,0 +1,25 @@
+package errors
+
+type ExitCode int
+
+const (
```

Comment:
Suggest we avoid collisions with nix-y things like sysexit.h and use 200+

Reply:
Good point, will do

Reply:
Good point, will do